{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 0, "total_error": 1}, "tests": {"0": {"status": "ERROR", "message": "request = <SubRequest 'setup' for <Function test_verify_observation_code>>\n\n    @pytest.fixture(scope=\"class\")\n    def setup(request):\n        print(\"setup\")\n        request.cls.allure_util = AllureReport(None)\n        request.cls.config = Config()\n        request.cls.generic_util = GenericUtils()\n        request.cls.observation_df = ExcelUtils().excel_to_df(fhir_requirement_path, observation_excel_sheet)\n        request.cls.non_observation_df = ExcelUtils().excel_to_df(fhir_requirement_path, non_observation_excel_sheet)\n        request.cls.fhir_json = read_fhir_json()\n        request.cls.docker_util = DockerUtils()\n        request.cls.dicom_util = DicomUtils()\n        request.cls.cxr_req = read_cxr_req()\n>       request.cls.exception_utils = ExceptionUtils()\nE       TypeError: ExceptionUtils.__init__() missing 1 required positional argument: 'log'\n\nconftest.py:25: TypeError\n", "test_name": "test_verify_observation_code", "rerun": "0"}}, "suite_name": "tests/cxr/fhir_validation/test_keys.py"}}}, "date": "February 01, 2024", "start_time": 1706780060.5749936, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "0", "skip": "0", "error": "1", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "1"}